scan-command/src/main/java/io/ballerina/scan/internal/ScanToolConstants.java

```diff
@@ -30,6 +30,7 @@ public class ScanToolConstants {
     static final String BALLERINA_ORG = "ballerina";
     static final String BALLERINAI_ORG = "ballerinai";
     static final String BALLERINAX_ORG = "ballerinax";
+    static final String USE_IMPORT_AS_SERVICE = " as _;";
```

Comment:
I would use

```suggestion
    static final String USE_IMPORT_AS_UNDERSCORE = " as _;";
```

It isn't just service, even in the scanner case, it is not.